enable-walletconnect.patch

```diff
@@ -0,0 +1,2197 @@
+diff --git a/package-lock.json b/package-lock.json
+index f6495db..379a54f 100644
+--- a/package-lock.json
++++ b/package-lock.json
```

Comment:
Sometimes the package lock syntax might change depending on the npm version. I think we should have here only the changes in the `package.json` file, then the package lock would be updated after it, in case the user decides to apply it.

What do you think?

Reply:
I agree! Fixed in d469ab4a66530b220f87a04d08a4c88a1f97204c